src/index.ts

```diff
@@ -419,7 +419,7 @@ export * from './util/locking/ExpiringReadWriteLocker';
 export * from './util/locking/EqualReadWriteLocker';
 export * from './util/locking/GreedyReadWriteLocker';
 export * from './util/locking/ReadWriteLocker';
-export * from './util/locking/RedisResourceLocker';
+export * from './util/locking/RedisLocker';
```

Comment:
The RELEASE_NOTES.md will have to be updated, both to mention the new feature and the change in class.